homework-2/types/UserAttrs.ts

```diff
@@ -1,4 +1,4 @@
-export type IUserAttrs = {
+export type UserAttrs = {
```

Comment:
There is a reason to use type instead of interface?

Reply:
I can't remember the main reason but maybe because of IIdParam interface looks not good=)